test/cache-interceptor/cache-stores.js

```diff
@@ -1,12 +1,20 @@
 'use strict'
 
+const { env, execArgv } = require('node:process')
 const { describe, test } = require('node:test')
 const { deepStrictEqual, notEqual, equal } = require('node:assert')
 const { once } = require('node:events')
 const MemoryCacheStore = require('../../lib/cache/memory-cache-store')
 
 cacheStoreTests(MemoryCacheStore)
 
+if (
+  (env.NODE_OPTIONS && env.NODE_OPTIONS.match(/experimental(-|_)sqlite/)) ||
```

Comment:
since this requires the experimental flag, what do we want to do for ci?

Reply:
What about adding a new matrix for `NODE_OPTIONS`, and using `matrix.include` and `matrix.exclude` to guard the experimental flag on specific Node.js versions?

GitHub docs happen to have a matrix example with `NODE_OPTIONS`: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#example-using-a-multi-dimension-matrix

Reply:
I'd say we can create a new matrix that focuses on features that requires experimental functionalities from Node (similar to what @SukkaW suggests).